pwndbg/gdblib/arch.py

```diff
@@ -9,7 +9,7 @@
 
 # TODO: x86-64 needs to come before i386 in the current implementation, make
 # this order-independent
-ARCHS = ("x86-64", "i386", "aarch64", "mips", "powerpc", "sparc", "arm", "armcm", "rv32", "rv64")
+ARCHS = ("x86-64", "i386", "aarch64", "mips", "powerpc", "sparc", "arm", "armcm", "riscv:rv32", "riscv:rv64", "riscv)
```

Comment:
I wonder why did we start with `rv32` and `rv64` in the first place. Perhaps they worked at some GDB version or GDB can report it as such?

Reply:
here is some interesting comment maybe: https://github.com/bminor/binutils-gdb/commit/895b7b4e4bed9eee9d4a3d55bee876055736bfd3

Reply:
@peace-maker Ping, since you worked on https://github.com/pwndbg/pwndbg/pull/1770 do you maybe know some details about the reported arch for riscv in GDB? Did they change it in some version or what?

Reply:
gdb used `riscv:rv64` as architecture string already when I was working on that PR. But since the other prefixes were ignored in some other arches in this list, I figured I'd do it for riscv as well. I didn't consider the plain `"riscv"` architecture, sorry.

```
pwndbg> set architecture
auto               i386:intel         i386:x64-32:intel  i386:x86-64:intel
i386               i386:x64-32        i386:x86-64        i8086
```